src/ParameterDistributions.jl

```diff
@@ -702,23 +700,13 @@ Returns a flattened variance of the distributions
 var(d::Parameterized) = var(d.distribution)
 var(d::Samples) = var(d.distribution_samples, dims = 2)
 function var(d::VectorOfParameterized)
-    d_dims = get_dimensions(d)
-    block_var = Array{Any}(undef, size(d_dims)[1])
-
-    for (i, dimension) in enumerate(d_dims)
-        block_var[i] = var(d.distribution[i])
-    end
-    return cat(block_var..., dims = 1)
+    block_var = var.(d.distribution)
+    return reduce(vcat, block_var)
 end
 
 function var(pd::ParameterDistribution)
-    d_dims = get_dimensions(pd)
-    block_var = Array{Any}(undef, size(d_dims)[1])
-
-    for (i, dimension) in enumerate(d_dims)
-        block_var[i] = var(pd.distribution[i])
-    end
-    return cat(block_var..., dims = 1) #build the flattened vector
+    block_var = var.(pd.distribution)
+    return reduce(vcat, block_var) #build the flattened vector
```

Comment:
broadcast `var` instead of looping. Then `reduce(vcat, ...)` instead of `cat(..., dims=1)`.